grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -33,7 +33,17 @@ class LoadDataService {
         CSVMapReader csvReader = new CSVMapReader(csvURL.newInputStream().newReader());
 
         ImportDataCommand command = new ImportDataCommand();
-        command.setData(csvReader.readAll());
+
+        List<Map<String, String>> csvItems = csvReader.readAll()
+
+        for (i in 0..< csvItems.size()) {
+            Map<String, String> currentItem = csvItems.get(i);
+
+            currentItem.put("stateOrProvince", currentItem.get("stateProvince"));
```

Comment:
I think it's easier to rename the column name in file from `stateProvince` to `stateOrProvince` to match the field on address.

Reply:
done